maps/biter_battles_v2/config.lua

```diff
@@ -6,7 +6,7 @@ local bb_config = {
 	["south_side_team_name"] = "Team South",
 
 	--TERRAIN OPTIONS--
-	["border_river_width"] = 44,						--Approximate width of the horizontal impassable river separating the teams. (values up to 64)
+	["border_river_width"] = 46,						--Approximate width of the horizontal impassable river separating the teams. (values up to 64)
```

Comment:
```suggestion
	["border_river_width"] = 44,						--Approximate width of the horizontal impassable river separating the teams. (values up to 64)
```
![image](https://github.com/Factorio-Biter-Battles/Factorio-Biter-Battles/assets/88335092/71bfcc67-6379-4029-ad26-fca427ed7048)
with this change river variates as in original at  y for north side: -26, -25, -24, -23

Reply:
Done

Reply:
Resolving it as you fixed it from what you said.